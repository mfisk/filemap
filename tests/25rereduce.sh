#!/bin/sh

fm -v map -i '/test/*' "split.sh < sort -n > sort -n -m | uniq -c" | sort -n
