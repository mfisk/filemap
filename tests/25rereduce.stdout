   1 1	2	8
   1 1	3	5
   2 1	2	3
   2 4	5	6
   2 7	8	9
